### PR TITLE
Apply templates

### PR DESCRIPTION
**Pull request by joao-p-marques**
Apply https://github.com/Tecnativa/image-template and https://github.com/copier-org/autopretty

The GH CI needs polishing, especially with regards to the built architectures. 

@Tecnativa
TT27819

### CODE REVIEW

File: .copier-answers.image-template.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Changes here will be overwritten by Copier; do NOT edit manually
+_commit: v0.1.3
+_src_path: https://github.com/Tecnativa/image-template.git
+dockerhub_image: ""
```
Comment (Yajo) [inline .copier-answers.image-template.yml:4]: I think that for this particular image we can still use the docker hub, as it has no backwards incompatibilities and is supported. This is not like docker-duplicity, where we needed to change the publishing structure.

**Review by Yajo — APPROVED** [review]

File: .copier-answers.image-template.yml (hunk 1 above)
Comment (joao-p-marques) [reply .copier-answers.image-template.yml:4]: Yes, I left this out for now as I think the current image on DockerHub is being built directly from the Dockerfile, as a responsibility of DockerHub, right? 
We'd need to disable that and then add it back here. Can you do that please?

Comment (Yajo) [reply .copier-answers.image-template.yml:4]: Done.

**Comment by joao-p-marques** [comment]
Changes made @Yajo

**Review by Yajo — APPROVED** [review]

**Pull request merged**